gapipy/resources/base.py

```diff
@@ -28,17 +28,17 @@ def __init__(self, data, stub=False, client=None):
     def options(cls, client):
         return APIRequestor(client, cls).options()
 
-    def fetch(self):
+    def fetch(self, httperrors_mapped_to_none=None):
         logger.info('Fetching %s/%s', self._resource_name, self.id)
```

Comment:
Lets add a docstring here; also referencing PR #119 & mirroring the change from version `2.25.0`

Reply:
done!